main/src/cgeo/geocaching/CachePopup.java

```diff
@@ -38,7 +40,11 @@ void showDialog() {
     @Override
     public void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        supportRequestWindowFeature(Window.FEATURE_NO_TITLE);
+        try {
+            supportRequestWindowFeature(Window.FEATURE_NO_TITLE);
+        } catch (final AndroidRuntimeException ex) {
+            Log.e("Error requesting indeterminate progress", ex);
```

Comment:
Shouldn't that log "Error requesting Window.FEATURE_NO_TITLE"?

Reply:
It should of course :)